build_scripts/performance_test.py

```diff
@@ -3,14 +3,15 @@
 import os
 import sys
 import time
+import glob
 import lipsum
 import pandas as pd
 from subprocess_calls import call_subprocess
 
 
 def create_input():
     """ Function used to generate the input file to do the performance test"""
-    row_size = 5  # given row size = 10 will generate around 17,000 rows of data in the excel file generated.
+    row_size = 3  # given row size = 10 will generate around 17,000 rows of data in the excel file generated.
```

Comment:
Can you please help justify row_size=3? Why not 1, or why not keep it at 5?

Reply:
Like mentioned in the comments, this is to generate the dynamic inputfile for the performance benchmark. with '5' the github action some times comes out as the there are number of files to be written back to disk after similarity analysis